### DISCUSSION TITLE
How to split a long equation and make them show in two lines

### DISCUSSION THREAD

**Discussion by XinyanYang95** [quarto-dev/quarto-cli#5550] [Q&A] [answered]
I have a long equation in .qmd. I want to make it show in two lines in both HTML and PDF. If I ender this .qmd to HTML. It displays in two lines. However, when I render it into PDF. An error was reported which told me \begin{multline} cannot work within ```$$...$$```.  Are there any solutions without removing ```$$```? 
```
$$
\begin{multline*}
-m_2 \ddua_2 - c_2 \dua_1 -c_2 \dua_2 - c_3 \dua_2 + c_3 \dua_3 \\
+ k_2 \ua_1 - k_2 \ua_2 - k_3 \ua_2 +k_3 \ua_3 = 0 \tag{b}
\end{multline*} 
$${#eq-eom3DOF}
```

**Comment by baptiste** [comment, marked as answer]
you can use a variant that's happy to live inside `$$` like `multlined` from mathtools,

````
---
title: "Untitled"
format: 
  pdf:
    keep-tex: true
    include-in-header: 
      - text: \usepackage{mathtools}
---

$$
\begin{multlined}
-m_2 a_2 - c_2 b_1 -c_2 b_2 - c_3 b_2 + c_3 b_3 \\
+ k_2 c_1 - k_2 c_2 - k_3 c_2 +k_3 c_3 = 0 
\end{multlined} 
$${#eq-eom3DOF}
````

**Comment by mcanouil** [comment]
See #2170
